src/templates/withDefaultContext.ts

```diff
@@ -12,7 +9,7 @@ export const withDefaultContext = (context: any = {}) => {
 
     const defaultContext: object = {
         // common var for translation
-        _: ((...args: any[]) => (context.__ ? context.__(...args) : String(args))),
+        __: ((...args: any[]) => (context.__ ? context.__(...args) : String(args))),
```

Comment:
Why suddenly two? I18n convention or other reason?

Reply:
It was in previous version and i18n it also uses